src/test/java/core/basesyntax/strategy/BalanceServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.strategy;
+
+import static core.basesyntax.db.Storage.storage;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.db.StorageImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.operation.BalanceService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class BalanceServiceTest {
+    private static BalanceService balanceService;
+    private static StorageImpl storageImpl;
+
+    @BeforeAll
+    static void beforeAll() {
+        storageImpl = new StorageImpl();
+        balanceService = new BalanceService(storageImpl);
+    }
+
+    @Test
+    void balanceService_negativeAmount_notOk() {
+        storage.put("apple", 20);
+        FruitTransaction fruitTransaction =
+                new FruitTransaction(FruitTransaction.Operation.BALANCE, "apple",-30);
```

Comment:
```suggestion
                new FruitTransaction(FruitTransaction.Operation.BALANCE, "apple", -30);
```

Check in all places (Ctrl + ALT + L).